apps/testclient/views.py

```diff
@@ -307,22 +308,28 @@ def authorize_link_v2(request):
 @never_cache
 @waffle_switch('enable_testclient')
 def authorize_link(request, v2=False):
-    pkce_enabled = request.GET.get('pkce')
-    request.session.update(test_setup(v2=v2, pkce=pkce_enabled))
+    # Always force pkce true
+    request.session.update(test_setup(v2=v2, pkce=True))
```

Comment:
I meant to leave a comment here too yesterday, maybe it just didn't save. I think we can remove the pkce parameter from `test_setup` and just always treat it as we would if pkce=True were set. Taking the mindset that even without specifying anything for PKCE, the default is that we use it and expect it to be used.